### ISSUE TITLE
Replace `conventional-commit` with `reno`

### ISSUE THREAD

**Issue by mnaser** [vexxhost/atmosphere#1080]
We're better off with following the same structure that exists within OpenStack such as `reno`.

**Comment by gtirloni** [comment]
@mnaser  is the plan to continue to use release-please for release PRs and other automation and reno for changelog generation?

**Comment by mnaser** [comment]
> @mnaser is the plan to continue to use release-please for release PRs and other automation and reno for changelog generation?

I don't think so, I'd like us to get in the habit of using `reno` for release notes and we can add those to our docs on every release.

**Comment by mnaser** [comment]
this is done :)
